test_project/rename_forward/urls.py

```diff
@@ -8,6 +8,7 @@
 class LinkedDataView(autocomplete.Select2QuerySetView):
     def get_queryset(self):
         qs = super(LinkedDataView, self).get_queryset()
+        print(self.forwarded.get('checkbox', None))
```

Comment:
Understood. Next time I make a pull request, and tests aren't as they should be, don't push them. :)